cloudinary/api_client/execute_request.py

```diff
@@ -70,12 +70,13 @@ def execute_request(http_connector, method, params, headers, auth, api_url, **op
         raise GeneralError("Unexpected error %s" % str(e))
     except socket.error as e:
         raise GeneralError("Socket Error: %s" % str(e))
+    except Exception as e:
+        raise GeneralError("API is only supported on dynamic folders please check your folder mode on  https://console.cloudinary.com/pm/developer-dashboard Error:%s " % str(e))
 
     try:
         result = json.loads(body.decode('utf-8'))
     except Exception as e:
-        # Error is parsing json
-        raise GeneralError("Error parsing server response (%d) - %s. Got - %s" % (response.status, body, e))
+        raise GeneralError("API is only supported on dynamic folders please check your folder mode on  https://console.cloudinary.com/pm/developer-dashboard Error:%s " % str(e))
```

Comment:
There can be different reasons to get a general error, not related to dynamic folders, this message will be misleading in all other use cases. We will open an issue for our backend team, the correct error should come back from our servers.

Reply:
ohh got it. Should I make any changes or wait? 

thanks

Reply:
@someshfengde you can close this PR, since it's not related to the SDK.
@tommyg-cld will pass the issue to the backend team.

Reply:
Thanks, 

got it :)